bin/mff2json.py

```diff
@@ -0,0 +1,52 @@
+"""
+Copyright 2019 Brain Electrophysiology Laboratory Company LLC
+
+Licensed under the ApacheLicense, Version 2.0(the "License");
+you may not use this module except in compliance with the License.
+You may obtain a copy of the License at:
+
+http: // www.apache.org / licenses / LICENSE - 2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ANY KIND, either express or implied.
+"""
+
+"""
+This script is used to export the contents of an MFF file into a JSON format.
+
+@author: Damian Persico - persicodamian@gmail.com
+"""
+
+from os.path import splitext
+from mffpy import Reader, Writer
+from argparse import ArgumentParser
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument("--mff_file", type=str, help="Path to an MFF directory")
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+    input_filename = parse_args().mff_file
+
+    if input_filename:
+        # Check .mff input
+        dir_and_base, ext = splitext(input_filename.rstrip('/'))
```

Comment:
Use `input_filename.rstrip(os.path.sep)` instead of `input_filename.rstrip('/')` so that it's platform independent